zeppelin/notebook/2G8S4616X/note.json

```diff
@@ -0,0 +1,528 @@
+      "user": "anonymous",
+      "dateUpdated": "2021-06-22 15:55:04.409",
+      "config": {
+        "colWidth": 12.0,
+        "fontSize": 9.0,
+        "enabled": true,
+        "results": {},
+        "editorSetting": {
+          "language": "scala",
+          "editOnDblClick": false,
+          "completionKey": "TAB",
+          "completionSupport": true
+        },
+        "editorMode": "ace/mode/scala"
+      },
+      "settings": {
+        "params": {},
+        "forms": {}
+      },
+      "results": {
+        "code": "ERROR",
+        "msg": [
+          {
+            "type": "TEXT",
+            "data": "java.lang.IllegalStateException: Cannot start query with id e6ec01cf-c5cf-4de0-bb76-51bf5332e1e6 as another query with same id is already active. Perhaps you are attempting to restart a query from checkpoint that is already active.\n  at org.apache.spark.sql.streaming.StreamingQueryManager.startQuery(StreamingQueryManager.scala:345)\n  at org.apache.spark.sql.streaming.DataStreamWriter.start(DataStreamWriter.scala:325)\n  ... 47 elided\n"
+          }
+        ]
+      },
+      "apps": [],
+      "jobName": "paragraph_1624377251418_248436506",
+      "id": "20210622-155411_325545705",
+      "dateCreated": "2021-06-22 15:54:11.418",
+      "dateStarted": "2021-06-22 15:55:04.435",
+      "dateFinished": "2021-06-22 15:55:05.294",
+      "status": "ERROR",
+      "progressUpdateIntervalMs": 500
+    },
+    {
+      "text": "# Run this one to make the stream stop\n\nquery.stop()",
+      "user": "anonymous",
+      "dateUpdated": "2021-06-22 15:54:11.299",
+      "config": {
+        "colWidth": 12.0,
+        "fontSize": 9.0,
+        "enabled": true,
+        "results": {},
+        "editorSetting": {
+          "language": "scala",
+          "editOnDblClick": false,
+          "completionKey": "TAB",
+          "completionSupport": true
+        },
+        "editorMode": "ace/mode/scala"
+      },
+      "settings": {
+        "params": {},
+        "forms": {}
+      },
+      "apps": [],
+      "jobName": "paragraph_1624377235674_-432074505",
+      "id": "20210622-155355_1080819727",
+      "dateCreated": "2021-06-22 15:53:55.674",
+      "status": "READY",
+      "progressUpdateIntervalMs": 500
+    },
+    {
+      "text": "%neo4j\n\nMATCH (n:Timestamp) RETURN COUNT(n) AS count",
```

Comment:
I'll take care of having the same psw across the whole source! Idk why but it worked for me, probably because I had an old version of the container that used an old password